components/desktop/gst-plugins-base1/gst-plugins-base1.p5m

```diff
@@ -0,0 +1,356 @@
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtsp-enumtypes.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtsp.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtspconnection.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtspdefs.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtspextension.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtspmessage.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtsprange.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtsptransport.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/gstrtspurl.h
+file path=usr/include/gstreamer-1.0/gst/rtsp/rtsp.h
+file path=usr/include/gstreamer-1.0/gst/sdp/gstmikey.h
+file path=usr/include/gstreamer-1.0/gst/sdp/gstsdp.h
+file path=usr/include/gstreamer-1.0/gst/sdp/gstsdpmessage.h
+file path=usr/include/gstreamer-1.0/gst/sdp/sdp.h
+file path=usr/include/gstreamer-1.0/gst/tag/gsttagdemux.h
+file path=usr/include/gstreamer-1.0/gst/tag/gsttagmux.h
+file path=usr/include/gstreamer-1.0/gst/tag/tag.h
+file path=usr/include/gstreamer-1.0/gst/tag/xmpwriter.h
+file path=usr/include/gstreamer-1.0/gst/video/colorbalance.h
+file path=usr/include/gstreamer-1.0/gst/video/colorbalancechannel.h
+file path=usr/include/gstreamer-1.0/gst/video/gstvideoaffinetransformationmeta.h
+file path=usr/include/gstreamer-1.0/gst/video/gstvideodecoder.h
+file path=usr/include/gstreamer-1.0/gst/video/gstvideoencoder.h
+file path=usr/include/gstreamer-1.0/gst/video/gstvideofilter.h
+file path=usr/include/gstreamer-1.0/gst/video/gstvideometa.h
+file path=usr/include/gstreamer-1.0/gst/video/gstvideopool.h
+file path=usr/include/gstreamer-1.0/gst/video/gstvideosink.h
+file path=usr/include/gstreamer-1.0/gst/video/gstvideoutils.h
+file path=usr/include/gstreamer-1.0/gst/video/navigation.h
+file path=usr/include/gstreamer-1.0/gst/video/video-blend.h
+file path=usr/include/gstreamer-1.0/gst/video/video-chroma.h
+file path=usr/include/gstreamer-1.0/gst/video/video-color.h
+file path=usr/include/gstreamer-1.0/gst/video/video-converter.h
+file path=usr/include/gstreamer-1.0/gst/video/video-dither.h
+file path=usr/include/gstreamer-1.0/gst/video/video-enumtypes.h
+file path=usr/include/gstreamer-1.0/gst/video/video-event.h
+file path=usr/include/gstreamer-1.0/gst/video/video-format.h
+file path=usr/include/gstreamer-1.0/gst/video/video-frame.h
+file path=usr/include/gstreamer-1.0/gst/video/video-info.h
+file path=usr/include/gstreamer-1.0/gst/video/video-multiview.h
+file path=usr/include/gstreamer-1.0/gst/video/video-overlay-composition.h
+file path=usr/include/gstreamer-1.0/gst/video/video-resampler.h
+file path=usr/include/gstreamer-1.0/gst/video/video-scaler.h
+file path=usr/include/gstreamer-1.0/gst/video/video-tile.h
+file path=usr/include/gstreamer-1.0/gst/video/video.h
+file path=usr/include/gstreamer-1.0/gst/video/videoorientation.h
+file path=usr/include/gstreamer-1.0/gst/video/videooverlay.h
+#file path=usr/lib/$(MACH64)/girepository-1.0/GstAllocators-1.0.typelib
```

Comment:
Worked in my limited testing (built gstreamer 1, launched some gst-launch-1.0, gst-typefind-1.0 and so on. Would be interesting to find out how to reproduce the issue with introspection.